### ISSUE TITLE
Search over all the fields

### ISSUE THREAD

**Issue by metaodi** [open-data/ckanext-scheming#77]
If the CKAN search should "know" about all the defined fields of a schema, this extensions should make use of the [`before_index`](http://docs.ckan.org/en/latest/extensions/plugin-interfaces.html?highlight=interfaces#ckan.plugins.interfaces.IPackageController.before_index) method and customize the dict that gets sent to Solr.

This issue is probably related to #54.

**Comment by wardi** [comment]
Yes, one problem is that sometimes fields defined in scheming need to change the solr schema. At the moment the best option is to customize schema.xml and before_index by hand.

In the future some automatic use of https://cwiki.apache.org/confluence/display/solr/Schema+API#SchemaAPI-ModifytheSchema would be *much* nicer.
